### PR TITLE
remove the MPL2.0 license pkg

### PR DESCRIPTION
**Pull request by CaiYueTing**
## Summary by CodeRabbit

- **Bug Fixes**
	- Simplified error handling in the message cloning process, improving clarity and performance when encountering issues.

- **Chores**
	- Removed unnecessary dependencies, leading to a cleaner and more efficient project structure.

### CODE REVIEW

File: message/pool/message.go [hunk 1]
```diff
@@ -617,13 +617,12 @@ func (r *Message) Clone(msg *Message) error {
 	}
 	_, err = io.Copy(buf, r.Body())
 	if err != nil {
-		var errs *multierror.Error
-		errs = multierror.Append(errs, err)
+		errStr := []string{err.Error()}
```
[the remark this answers is not in the document]
Comment (jkralik) [reply message/pool/message.go:620]:
```go
	var errs []error
	_, errS := r.Body().Seek(n, io.SeekStart)
	if errS != nil {
	    errStr = append(errs, errS)
	}
	return errors.Join(errs...)
```

[the remark this answers is not in the document]
Comment (CaiYueTing) [reply message/pool/message.go:620]: I don't understand. It's just a copy error and a seek error that happened there.

[the remark this answers is not in the document]
Comment (CaiYueTing) [reply message/pool/message.go:620]: ok, i try this again

[the remark this answers is not in the document]
Comment (CaiYueTing) [reply message/pool/message.go]: @jkralik Thank you for your review, your way looks like good  to me.

**Comment by Danielius1922** [comment]
@jkralik maybe we should add the licence scanner (fossa) that we have in other repositories here, hopefully it can be setup in such way to mark MPL as problematic. Because I'll forget about it in a week and we use the multierror package in other repositories, so I can easily see myself re-adding it in the future. 🤓

**Pull request merged**